dotm_search.py

```diff
@@ -1,15 +1,50 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
+import os
+import argparse
+import zipfile
+parser = argparse.ArgumentParser()
+
 
 """
 Given a directory path, search all files in the path for a given text string
 within the 'word/document.xml' section of a MSWord .dotm file.
 """
-__author__ = "???"
+__author__ = "knmarvel"
+#   time spent on this project: 2 hrs
+
+
+def examine_files(dir, char):
+    files = {}
+    total_files_searched = 0
+    for filename in os.listdir(dir):
+        filepath = dir + "/" + filename
+        if zipfile.is_zipfile(filepath):
+            with zipfile.ZipFile(filepath, 'r') as zip_ref:
+                zip_ref.extractall("./extracted")
+            with open('./extracted/word/document.xml', "r") as doc:
+                doc = doc.read()
+            if "office/word" in doc:
```

Comment:
where did "office/word" come from?

Reply:
The header of the document file. I could have used more of it, like, w:document xmlns:wpc="http://schemas.microsoft.com/office/word/2010/wordprocessingCanvas", but thought that was overkill.